backend/layers/api/portal_api.py

```diff
@@ -147,14 +148,24 @@ def remove_none(body: dict):
 
 
 # Note: `metadata` can be none while the dataset is uploading
-def _dataset_to_response(dataset: DatasetVersion, is_tombstoned: bool, is_in_published_collection: bool = False):
+def _dataset_to_response(
+    dataset: DatasetVersion, is_tombstoned: bool, is_in_revision: bool = False, revision_created_at: datetime = None
+):
     dataset_id = dataset.version_id.id
     # Only return `dataset_deployments` if a CXG artifact is available. This is to prevent the "Explore"
     # button to show up while a dataset upload is in progress
     if any(a for a in dataset.artifacts if a.type == DatasetArtifactType.CXG):
-        dataset_deployments = [{"url": explorer_url.generate(dataset, is_in_published_collection)}]
+        use_canonical = False if is_in_revision else True
```

Comment:
Technically this could be: `use_canonical = not is_in_revision`, but I am also ok with the explicit form.